dfa_identify/graphs.py

```diff
@@ -27,8 +27,8 @@ class APTA:
     """Augmented Prefix Tree Acceptor."""
     tree: nx.DiGraph
     alphabet: bidict  # Mapping from token to int.
-    ord_prefs: set[Tuple[Node, Node]]
-    inc_prefs: set[Tuple[Node, Node]]  # MemReP 'incomparable' preferences (incomparable_word_1, incomparable_word_2)
+    num_ord_prefs: int
+    num_inc_prefs: int
```

Comment:
I actually think I prefer not putting the preferences into the `APTA`, if its not too much work.

The previous definition seems like a better match and more direct.